### DISCUSSION TITLE
Pagina pasteleria (simple)

### DISCUSSION THREAD

**Discussion by davidtoString** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#233] [Show and tell]
Aprovecho para mostrarles como quedó la mía. 

Es la versión simple, pues tiene espacio para mejoras en el estilo, en algunos graficos y en la zona privada para pasteleros, pero eso lo haré más adelante. Me apoyé de algunos componentes de bootstrap para darle un formato mas prolijo. 

Aqui el enlace:  https://davidmartinz.github.io/LaunchX-Semana2/index.html 

✌️

**Comment by hackit92** [comment]
Excelente trabajo, creo que cumple bastante bien los requerimientos de la práctica.

**Comment by JuanSalvador73** [comment]
No solo cumple los requisitos, los rebasa. Buen trabajo :D

**Comment by charlyguz** [comment]
Muy buen trabajo la verdad
